doc/rst/usingchapel/chplenv.rst

```diff
@@ -197,18 +198,22 @@ CHPL_*_COMPILER
    The default for ``CHPL_*_COMPILER`` depends on the value of the corresponding
    ``CHPL_*_PLATFORM`` environment variable:
 
-        ============  ==================================================
-        Platform      Compiler
-        ============  ==================================================
-        cray-x*       - gnu (for ``CHPL_HOST_COMPILER``)
-                      - cray-prgenv-$PE_ENV (for ``CHPL_TARGET_COMPILER``,
-                        where PE_ENV is set by PrgEnv-* modules)
-        darwin        clang if available, otherwise gnu
-        pwr6          ibm
-        other         gnu
-        ============  ==================================================
-
-   If ``CHPL_HOST_PLATFORM == CHPL_TARGET_PLATFORM`` and is not ``cray-x*``,
+        +-------------+------------------------------------------------------+
```

Comment:
I don't think we use this style table in other rst files. Any reason for the change?

Reply:
Yeah, I can't get row spanning for "cray-xc, hpe-cray-xc" without doing the full syntax, and if I don't row-span that first entry the second column gets squeezed really badly and looks awful.

Reply:
👍